### PR TITLE
Add separate routers for user & admin.

### PR DESCRIPTION
**Pull request by hetjagani**
Run separate servers for admin (internal use) & user (user facing).

### CODE REVIEW

File: action/membership/routes.go [hunk 1]
```diff
@@ -7,8 +7,8 @@ type membership struct {
 	PlanID uint `json:"plan_id" validate:"required"`
 }
 
-// Router - Group of membership router
-func Router() chi.Router {
+// UserRouter - Group of membership router
+func UserRouter() chi.Router {
 	r := chi.NewRouter()
 
 	r.Get("/", list)    // GET /memberships - return list of memberships
```
Comment (shreeharsha-factly) [inline action/membership/routes.go:14]: UserRouter must access *GET /memberships/{membership_id}* only

File: action/order/routes.go [hunk 2]
```diff
@@ -4,8 +4,8 @@ import (
 	"github.com/go-chi/chi"
 )
 
-// Router - Group of order router
-func Router() chi.Router {
+// UserRouter - Group of order router
+func UserRouter() chi.Router {
 	r := chi.NewRouter()
 
 	r.Get("/", list)    // GET /orders - return list of orders
```
Comment (shreeharsha-factly) [inline action/order/routes.go:11]: UserRouter must access *GET /orders/{order_id}* also

File: action/payment/routes.go [hunk 3]
```diff
@@ -13,8 +13,8 @@ type payment struct {
 	RazorpaySignature string `gorm:"column:razorpay_signature" json:"razorpay_signature" validate:"required"`
 }
 
-// Router - Group of payment router
-func Router() chi.Router {
+// UserRouter - Group of payment router
+func UserRouter() chi.Router {
 	r := chi.NewRouter()
 
 	r.Get("/", list)    // GET /payments - return list of payments
```
Comment (shreeharsha-factly) [inline action/payment/routes.go:20]: UserRouter must access *GET /payments/{payment_id}* also

File: action/membership/routes.go (hunk 1 above)
Comment (hetjagani) [reply action/membership/routes.go:14]: user should be able to create membership also

File: action/order/routes.go (hunk 2 above)
Comment (hetjagani) [reply action/order/routes.go:11]: It is already there at line 15

File: action/payment/routes.go (hunk 3 above)
Comment (hetjagani) [reply action/payment/routes.go:20]: It is already there at line 24

**Review by shreeharsha-factly — APPROVED** [review]

**Pull request merged**